src/internal_events/mod.rs

```diff
@@ -69,13 +80,12 @@ pub use self::statsd::*;
 pub use self::stdin::*;
 pub use self::syslog::*;
 pub use self::tcp::*;
-pub use self::udp::*;
 pub use self::unix::*;
 pub use self::vector::*;
 #[cfg(feature = "wasm")]
 pub use self::wasm::*;
 
-pub trait InternalEvent: std::fmt::Debug {
```

Comment:
Why is `Debug` requirement removed?

Reply:
I did not intend to leave in this change. However, this trait requirement is not needed for the current code base, so maybe the change should stand. Is there a reason to keep it in?

Reply:
I think it's here to be future proof. Not sure for what thou.  
cc. @lukesteensen

Reply:
I'm ok to remove it since it's unused. No reason to constrain things unnecessarily.